### ISSUE TITLE
The digital_ocean_tag module has stopped working

### ISSUE THREAD

**Issue by jdickey** [jdickey/docker-ansible-test#11]
Per ansible/ansible#33459, this is apparently due to the internal API now requiring the resource ID/Droplet ID to be sent as a string rather than as a number. (why do that?!?)

Two workarounds present themselves:

1. Instead of using the `digital_ocean_tag` module in a Playbook, invoke the `doctl` command-line utility; e.g., `doctl compute droplet tag "87654321" --tag-name "in_process"`. That has been shown to work while using Ansible 2.3.2.0 (against which the Ansible issue referenced above was opened); or
2. Upgrade Ansible (currently at [2.4.2.0](https://pypi.python.org/pypi/ansible/2.4.2.0); latest 2.3.*x* release is [2.3.3.0](https://pypi.python.org/pypi/ansible/2.3.3.0)).

**Comment by mitpaladin** [comment]
Does upgrading Ansible break anything else?

**Comment by mitpaladin** [comment]
Looks like the PR to fix the ID type was merged into ansible:devel, for 2.5.0.  Does not look like it is in 2.4.2.0?

**Comment by BondAnthony** [comment]
You can work around this issue by placing the 2.5 digital_ocean_tag module into the `library` directory in your project. This bug fix has been included into 2.4.4 release also.

**Comment by mitpaladin** [comment]
ah, ok, though guess you mean devel branch, which will be 2.5? https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/digital_ocean/digital_ocean_tag.py

**Comment by BondAnthony** [comment]
Yes this code is in 2.5 (devel) but it has also been cherry picked into the 2.4.4 (stable) release. Stable-2.4.4 will be a few weeks before release candidates are created.
